deployed-hmtoken/hmt.address.json

```diff
@@ -0,0 +1,3 @@
+{
+  "address": "0x4C18A2E51edC5043e9c4B6b0757990A4Ac13797f"
+}
```

Comment:
I dont think we want this checked in do we? this is an artifact of the token deploy

Reply:
Okay, removing this.